x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -153,63 +155,39 @@ export default function ({
         await unloadEcommerce();
       });
 
-      it('PNG file matches the baseline: small dashboard', async function () {
+      // Failing: See https://github.com/elastic/kibana/issues/142484
+      it.skip('PNG file matches the baseline: large dashboard', async function () {
         this.timeout(300000);
 
         await PageObjects.common.navigateToApp('dashboard');
-        await PageObjects.dashboard.loadSavedDashboard('Ecom Dashboard');
+        await PageObjects.dashboard.loadSavedDashboard('Large Dashboard');
         await PageObjects.reporting.openPngReportingPanel();
         await PageObjects.reporting.forceSharedItemsContainerSize({ width: 1405 });
         await PageObjects.reporting.clickGenerateReportButton();
         await PageObjects.reporting.removeForceSharedItemsContainerSize();
 
-        const url = await PageObjects.reporting.getReportURL(60000);
+        const url = await PageObjects.reporting.getReportURL(200000);
         const reportData = await PageObjects.reporting.getRawPdfReportData(url);
-        const reportFileName = 'small_dashboard_preserve_layout';
+        const reportFileName = 'large_dashboard_preserve_layout';
         const sessionReportPath = await PageObjects.reporting.writeSessionReport(
           reportFileName,
           'png',
           reportData,
           REPORTS_FOLDER
         );
-
-        const percentDiff = await png.compareAgainstBaseline(
-          sessionReportPath,
-          PageObjects.reporting.getBaselineReportPath(reportFileName, 'png', REPORTS_FOLDER),
-          REPORTS_FOLDER,
-          updateBaselines
-        );
-
-        expect(percentDiff).to.be.lessThan(0.09);
-      });
-
-      it('PNG file matches the baseline: large dashboard', async function () {
```

Comment:
Removed this test because it is redundant with a test under `Sample data from Kibana 7.6`